numbertest1.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime
+from functools import reduce
+
+
+# def double(x):
+#   return 2 * x
+
+
+# numbers = list(range(1, 10))
+# total = reduce(lambda accumulator, number: accumulator + number, numbers, 0)
+# print(total == 45)
+
+# [double(number) for number in numbers]
```

Comment:
Please remove comments:
```suggestion
 def double(x):
   return 2 * x


 numbers = list(range(1, 10))
 total = reduce(lambda accumulator, number: accumulator + number, numbers, 0)
 print(total == 45)

 [double(number) for number in numbers]
```

Reply:
Fixed it.